linker/shared/src/main/scala/org/scalajs/linker/checker/IRChecker.scala

```diff
@@ -185,10 +187,7 @@ private final class IRChecker(unit: LinkingUnit, logger: Logger) {
 
     classDef.kind match {
       case ClassKind.NativeJSClass | ClassKind.NativeJSModuleClass =>
-        if (classDef.jsNativeLoadSpec.isEmpty) {
-          reportError(
-              i"Native JS type ${classDef.name} must have a jsNativeLoadSpec")
-        }
+        () // may or may not have a native load spec
```

Comment:
While you are here, maybe add `isJSNativeClass` to `ClassKind`, use it in `Analyzer` (it already has that locally) and here?

Reply:
I'm not sure. There is nothing in common between a `NativeJSClass` and a `NativeJSModuleClass` other than having a `jsNativeLoadSpec`. It's not a very useful test. I already have trouble remembering which test does what, I'd rather not have another one which is not even that widespread.